Add instructions to avoid confusing error (#456)
DataDog/integrations-extras: eventstore/README.md

To install the EventStore check on your host:

1. Install the [developer toolkit][1] on any machine.

> 2. Make sure the repository used is set to integrations-extras where the EventStore integration lives: `ddev config set repo extras`

Reviewer: ```suggestion
2. Make sure the repository used is set to integrations-extras where the EventStore integration lives: `ddev config set extras ./<PATH_TO_THE_INTEGRATIONS_EXTRAS_REPO>`
```

> 3. Run `ddev release build eventstore` to build the package.

Reviewer: ```suggestion
3. Run `ddev -e release build eventstore` to build the package.
```

4. [Download the Datadog Agent][2].

> 5. Upload the build artifact to any host with an Agent and run `datadog-agent integration install -w path/to/eventstore/dist/<ARTIFACT_NAME>.whl`.

Reviewer: ```suggestion
5. Upload the build artifact to any host with an Agent and run `datadog-agent integration install -w <PATH_OF_INTEGRATION_NAME_PACKAGE>/<ARTIFACT_NAME>.whl`.
```
